CHANGELOG.md

```diff
@@ -5,6 +5,157 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+* Fix AutoencoderKL docstrings. (#8445)
+* Inverse Threading Fix (#8418)
+* Fix normalize intensity (#8286)
+* Fix path at test onnx trt export (#8361)
+
+### Changed
+* [DOC] Update README.md (#8157)
+* Streamlined Rearrange in SpatialAttentionBlock (#8130)
+* Optimize VISTA3D (#8123)
+* Skip torch trt convert test with torch newer than or equal to 2.5.0 (#8165)
+* Enable redirection of all loggers by configuring a FileHandler within the bundle (#8142)
+* Apply pyupgrade fixes for Python 3.9+ syntax (#8150)
+* Update base image to 2410 (#8164)
+* TRT support for MAISI (#8153)
+* 8134 Add unit test for responsive inference (#8146)
+* SwinUNETR refactor to accept additional parameters (#8212)
+* Allow an arbitrary mask to be used in the self attention (#8235)
+* Bump codecov/codecov-action from 4 to 5 (#8245)
+* Docs: update brats classes description (#8246)
+* Change default value of `patch_norm` to False in `SwinUNETR` (#8249)
+* Modify Dice, Jaccard and Tversky losses (#8138)
+* Modify Workflow to Allow IterableDataset Inputs (#8263)
+* Enhance download_and_extract (#8216)
+* Relax gpu load check (#8282, #8275)
+* Using LocalStore in Zarr v3 (#8299)
+* Enable gpu load nifti (#8188)
+* update pydicom reader to enable gpu load (#8283)
+* Zarr compression tests only with versions before 3.0 (#8319)
+* Changing utils.py to test_utils.py (#8335)
+* Refactor testd (#8231)
+* Recursive Item Mapping for Nested Lists in Compose  (#8187)
+* Bump min torch to 1.13.1 to mitigate CVE-2022-45907 unsafe usage of eval (#8296)
+* Inferer modification - save_intermediates clashes with latent shape adjustment in latent diffusion inferers (#8343)
+* Solves path problem in test_bundle_trt_export.py (#8357)
+* Modify ControlNet inferer so that it takes in context when the diffus… (#8360)
+* Update monaihosting download method (#8364)
+* Bump torch minimum to mitigate CVE-2024-31580 & CVE-2024-31583 and enable numpy 2 compatibility (#8368)
+* Auto3DSeg algo_template hash update (#8378)
+* Enable Pytorch 2.6 (#8309)
+* Auto3DSeg algo_template hash update (#8393, #8397)
+* Update Dice Metric Docs (#8388)
+* Auto3DSeg algo_template hash update (#8406)
+* Update bundle download API (#8403)
+* Add Skip test in TestTranschex (#8416)
+* Update get latest bundle version function (#8420)
+* Temporarily Restrict setuptools Version to 79.0.1 (#8441)
+* Update default overlap value in occlusion_sensitivity to 0.6 (#8446)
+* Enable code coverage comments on PRs in codecov configuration (#8402)
+* Migrate to modern Python Logger API (#8449)
+
+### Deprecated
+### Removed
+* Remove deprecated functionality for v1.5 (#8430)
+* Remove deprecated `return_state_dict ` in bundle `load` (#8454)
+* Remove deprecated `net_name` in test file (#8461)
+* Remove unused test cases in bundle load (#8463)
+* selfattention block: Remove the fc linear layer if it is not used (#8325)
+* Removed outdated `torch` version checks from transform functions (#8359)
+
+## New Contributors
```

Comment:
We don't need add these info in the changelog, we can only include it in the release note, what do you think?